advanced/spec/hashTable_test.js

```diff
@@ -0,0 +1,11 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import HashTable from '../src/hashTable.js'
+
+describe('HashTable', () => {
+
+  it('exists', () => {
+    expect(HashTable).to.be.a('function')
+  })
+
```

Comment:
How do you know if this works if you don't have tests? Are you testing manually or something?